client/src/pages/ModalBookingScreen.js

```diff
@@ -33,12 +33,18 @@ function ModalBookingScreen({ bookingDate, refreshBooking, desks }) {
 		value: desk.id,
 	}));
 
+	const testUser = users.map((e) => ({
```

Comment:
testUser name is not very descriptive name. userNameOptions would be a better name or any other you might think of

Reply:
Great spot! I had already changed it actually to something similar to that, but I just forgot to push it.